src/scanoss/ui/template/scanoss.html.twig

```diff
@@ -0,0 +1,26 @@
+{#
+    SPDX-License-Identifier: GPL-2.0
+    SPDX-FileCopyrightText: 2021 Sarita Singh <saritasingh.0425@gmail.com>  
+
+   This program is free software; you can redistribute it and/or 
+   modify it under the terms of the GNU General Public License 
+   version 2 as published by the Free Software Foundation.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of 
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+   GNU General Public License for more details. 
+
+   You should have received a copy of the GNU General Public License along
+   with this program; if not, write to the Free Software Foundation, Inc.,
+   51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#}
+{% import "include/macros.html.twig" as macro %}
+
+<li>
+  {{ 'ScanCode Toolkit'|trans }}<img src="images/info_16.png" title="{{ 'To scan code and detect licenses, copyrights, author and more.'|trans }}" alt="" class="info-bullet"/>, {{'scan for'|trans }}<br/>
+  <input type="checkbox" class="browse-upload-checkbox view-license-rc-size" name="scancodeFlags[]" value="license" /> {{ 'License'|trans }}<img src="images/info_16.png" title="{{ 'A full comparison between the database of licenses present in ScanCode and the code uploaded by users based approach'|trans }}" alt="" class="info-bullet"/><br />
+  <input type="checkbox" class="browse-upload-checkbox view-license-rc-size" name="scancodeFlags[]" value="copyright" /> {{ 'Copyright'|trans }}<img src="images/info_16.png" title="{{ 'Copyright along with copyright holder/author information'|trans }}" alt="" class="info-bullet"/><br />
+  <input type="checkbox" class="browse-upload-checkbox view-license-rc-size" name="scancodeFlags[]" value="email" /> {{ 'Email'|trans }}<img src="images/info_16.png" title="{{ 'Email(s) present in the scan code file'|trans }}" alt="" class="info-bullet"/><br />
+  <input type="checkbox" class="browse-upload-checkbox view-license-rc-size" name="scancodeFlags[]" value="url" /> {{ 'URL'|trans }} <img src="images/info_16.png" title="{{ 'URL(s) present in the scan code file'|trans }}" alt="" class="info-bullet"/>
```

Comment:
Please add `data-toggle="tooltip"` attribute to all tool-tip images for bootstrap.